Mage/EventManager.cs

```diff
@@ -3,38 +3,53 @@
 using Wizcorp.MageSDK.Event;
 using Wizcorp.MageSDK.Log;
 
-namespace Wizcorp.MageSDK.MageClient {
-	public class EventManager : EventEmitter<JToken> {
-		private Mage mage { get { return Mage.Instance; } }
-		private Logger logger { get { return mage.logger("eventManager"); } }
+namespace Wizcorp.MageSDK.MageClient
+{
+	public class EventManager : EventEmitter<JToken>
+	{
+		private Mage Mage
+		{
+			get { return Mage.Instance; }
+		}
+
+		private Logger Logger
+		{
+			get { return Mage.Logger("eventManager"); }
```

Comment:
you mean the logger context?
thats a tricky one as we use camel case on the BE
however technically this is a client context
lets discuss and define a standard for this CC: @nullorvoid @ronkorving